pytools/version.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (2020, 4, 3)
+VERSION = (2020, 4, 4)
```

Comment:
```suggestion
VERSION = (2020, 4, 5)
```
(2020.4.4 was just released)